packages/snjs/lib/models/app/userPrefs.ts

```diff
@@ -31,7 +33,9 @@ export type PrefValue = {
   [PrefKey.SortNotesBy]: CollectionSort;
   [PrefKey.SortNotesReverse]: boolean;
   [PrefKey.NotesShowArchived]: boolean;
+  [PrefKey.NotesShowDeleted]: boolean;
```

Comment:
Hmm looks like we already have something similar? https://github.com/standardnotes/snjs/pull/457/files#diff-bbe8e7705d98fb9ec1181e74bc0c8bb20e12c3ac1b8596494c6c56a696de40fcR101

Reply:
Line 101
<img width="724" alt="Screen Shot 2021-10-17 at 10 11 53 AM" src="https://user-images.githubusercontent.com/3277844/137633412-456e80a8-44b6-471f-9331-50b3c3727144.png">

Reply:
Ok, let's rename this to Show Trashed at the very least, as Deleted is a separate concept that is permanent and irrecoverable.

Reply:
The design says "Show deleted notes". I guess I should change that as well?